Dockerfile.prod

```diff
@@ -0,0 +1,17 @@
+FROM node:18 AS builder
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install --ignore-platform
+COPY . .
+RUN yarn build
+
+# 2nd Stage
+FROM nginx:1.14.2-alpine
+
+COPY --from=0 /usr/src/app/build /usr/share/nginx/html
+WORKDIR /usr/share/nginx/html
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
when would we be using this docker file? seeing this repo  uses github pages to deploy the site

Reply:
Okay I got it, delete unnecessary files.